common/tier4_autoware_utils/include/tier4_autoware_utils/geometry/geometry.hpp

```diff
@@ -731,6 +735,70 @@ geometry_msgs::msg::Pose calcInterpolatedPose(
 
   return output_pose;
 }
+
+inline geometry_msgs::msg::Vector3 createVector3(const double x, double y, double z)
+{
+  return geometry_msgs::build<geometry_msgs::msg::Vector3>().x(x).y(y).z(z);
+}
+
+inline geometry_msgs::msg::Twist getTwist(
```

Comment:
To be the consistent naming style.

```suggestion
inline geometry_msgs::msg::Twist createTwist(
```